specials/SPC18_Matias_SpiesVsMercs.Altis/victory_handler.sqf

```diff
@@ -0,0 +1,17 @@
+//Spies win end
+if (spies_extracted == true) then
+{
+	"end2" call BIS_fnc_endMission;
```

Comment:
```suggestion
	"end2" call BIS_fnc_endMissionServer;
```

[BIS_fnc_endMission](https://community.bistudio.com/wiki/BIS_fnc_endMission) has local effect, it won't be propagated over the network, see the LE icon at the top of the documentation.

Reply:
Done for both ends